java/ql/test/experimental/query-tests/security/CWE-927/SensitiveBroadcast.java

```diff
@@ -0,0 +1,130 @@
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+class SensitiveBroadcast {
```

Comment:
Tests to add:

* At least one of the android-specific sensitive variable names (email, ticket etc)
* Use of Bundle's put methods (the existing tests all use Intent's put methods)
* Passing a non-empty array to one of the methods where the permission-set needs to be nonempty to be secure.

Reply:
I've modified existing tests and added some new tests to:
- Have some test cases use android-specific sensitive variable names (email and ticket)
- Use Bundle's put methods in three test cases (two <code>BAD</code> ones and one <code>GOOD</code>)
- Pass a non-empty array to the test case <code>sendBroadcast10</code> to confirm it passes the test thus it's secure